include/h2o.h

```diff
@@ -1234,6 +1234,12 @@ extern const char *h2o_http2_npn_protocols;
 extern const char *h2o_npn_protocols;
 extern const h2o_iovec_t *h2o_http2_alpn_protocols;
 extern const h2o_iovec_t *h2o_alpn_protocols;
+/**
+ * release all resources in tls used by h2o
+ * normally there is no need to do this like standalone server.
+ * but when used as a library(libh2o or libh2o-evloop), the user may want to release resources hold in tls
+ */
+void h2o_tls_data_dispose(void);
```

Comment:
How about renaming the function to `h2o_cleanup_thread`?

I think that we should avoid using "tls" since it is collides with the acronym for Transport Layer Security. I think we should use "cleanup" instead of "dispose" for two reasons: 1) in H2O "dispose" is used for releasing resources allocated to a particular object whereas in this case we are not taking an argument to an object 2) "cleanup" is the term used by pthread (e.g. `pthread_cleanup_push`).

Reply:
yes, fixed